be/src/vec/functions/function_case.h

```diff
@@ -332,13 +332,15 @@ class FunctionCase : public IFunction {
         bool then_null = false;
         for (int i = 1 + has_case; i < arguments.size() - has_else; i += 2) {
             auto then_column_ptr = block.get_by_position(arguments[i]).column;
-            if (then_column_ptr->is_nullable() || then_column_ptr->only_null()) {
+            if (then_column_ptr->is_nullable() ||
```

Comment:
maybe we can remove `then_column_ptr->is_nullable()`?

Reply:
It seems to be ok, for `ColumnConst`, it seems that `get_data_type() == TypeIndex::Nullable` is the real `is_nullable`